src/slurm_plugin/computemgtd.py

```diff
@@ -66,12 +67,13 @@ def _get_config(self, config_file_path):
         config = ConfigParser()
         try:
             # Use subprocess based method to copy shared file to local to prevent hanging when NFS is down
+            tmp_config_path = f"{SLURM_PLUGIN_DIR}/.computemgtd_config.local"
```

Comment:
the dir used as temporary needs to be on the local fs and not on an nfs share otherwise we don't address the NFS hanging problem

Reply:
ah, makes sense. fixing now.

Reply:
done.